Products/ZenEvents/tests/test_zeneventd.py

```diff
@@ -39,14 +40,14 @@ def setUp(self):
             created_time=1523044529575,
             event_class="/ZenossRM",
             actor=EventActor(
-                element_type_id=10,
+                element_type_id=ModelElementType.DEVICE,
                 element_uuid="575bcf2d-8ca0-47d9-8e63-b4f2c1242ef3",
                 element_identifier="device.loc",
-                element_sub_type_id=100,
+                element_sub_type_id=ModelElementType.COMPONENT,
                 element_sub_identifier="zeneventd",
             ),
             summary='Event Summary',
-            severity=1,
+            severity=EventSeverity.SEVERITY_DEBUG,
```

Comment:
While 1 _is_ a valid enumeration of `EventSeverity`, the code was changed to `EventSeverity.SEVERITY_DEBUG` to demonstrate that we should be using the symbolic names of values rather than the values directly.